dependency_support/load_external.bzl

```diff
@@ -295,12 +295,13 @@ def load_external_repositories():
         urls = ["https://github.com/grailbio/bazel-compilation-database/archive/940cedacdb8a1acbce42093bf67f3a5ca8b265f7.tar.gz"],
     )
 
-    # Tagged 2024-06-19, current as of 2024-06-26
+    # Tagged 2024-08-23, current as of 2024-08-24
+    VERIBLE_TAG = 'v0.0-3756-gda9a0f8c'
```

Comment:
Looks like bazel [`buildifier`](https://github.com/bazelbuild/buildtools/blob/main/buildifier/README.md) prefers double quotes here (and below the 'v').

We should add running `buildifier` in the CI on BUILD and *.bzl files...

Reply:
Done on the double quotes.